gpu-operator/dra-intro-install.rst

```diff
@@ -0,0 +1,121 @@
+
+To understand and make best use of NVIDIA's DRA Driver for GPUs, we recommend becoming familiar with DRA by working through the `official documentation <https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/>`_.
+
+
+The twofold nature of this driver
+=================================
+
+NVIDIA's DRA Driver for GPUs is comprised of two subsystems that are largely independent of each other: one manages GPUs, and the other one manages ComputeDomains.
+
+Below, you can find instructions for how to install both parts or just one of them.
+Additionally, we have prepared two separate documentation chapters, providing more in-depth information for each of the two subsystems:
+
+- :ref:`Documentation for ComputeDomain (MNNVL) support <dra_docs_compute_domains>`
+- :ref:`Documentation for GPU support <dra_docs_gpus>`
+
+
+************
+Installation
+************
+
+Prerequisites
+=============
+
+- Kubernetes v1.32 or newer.
+- DRA and corresponding API groups must be enabled (`see Kubernetes docs <https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#enabling-dynamic-resource-allocation>`_).
+- NVIDIA GPU Driver 565 or later.
+- While not strictly required, we recommend using NVIDIA's GPU Operator v25.3.0 or later, installed with CDI enabled (use the ``--set cdi.enabled=true`` commandline argument during ``helm install``). For reference, please refer to the GPU Operator `installation documentation <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#common-chart-customization-options>`__.
+
+..
+  For convenience, the following example shows how to enable CDI upon GPU Operator installation:
+  .. code-block:: console
+      $ helm install --wait --generate-name \
+          -n gpu-operator --create-namespace \
+          nvidia/gpu-operator \
+          --version=${version} \
+          --set cdi.enabled=true
+
+.. note::
+
+  If you want to use ComputeDomains and a pre-installed NVIDIA GPU Driver:
+
+  - Make sure to have the corresponding ``nvidia-imex-*`` packages installed.
+  - Disable the IMEX systemd service before installing the GPU Operator.
+  - Refer to the `docs on installing the GPU Operator with a pre-installed GPU driver <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#pre-installed-nvidia-gpu-drivers>`__.
+
+
+Configure and Helm-install the driver
+=====================================
+
+#. Add the NVIDIA Helm repository:
+
+   .. code-block:: console
+
+      $ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia \
+          && helm repo update
+
+#. Install the DRA driver, providing install-time configuration parameters.
+
+   Example for *Operator-provided* GPU driver:
+
+   .. code-block:: console
+
+      $ helm install nvidia-dra-driver-gpu nvidia/nvidia-dra-driver-gpu \
+          --version="25.3.0-rc.4" \
+          --create-namespace --namespace nvidia-dra-driver-gpu \
```

Comment:
```suggestion
          --create-namespace \
          --namespace nvidia-dra-driver-gpu \
```

Reply:
I don't like having more than one flag on a single line (unless they are all one exactly one single line). It makes it hard to scan top to bottom and see all of the flags.